schema.sql

```diff
@@ -45,3 +45,22 @@ ALTER TABLE animals DROP COLUMN IF EXISTS species;
 ALTER TABLE animals ADD COLUMN species_id INTEGER REFERENCES species(id);
 
 ALTER TABLE animals ADD COLUMN owner_id INTEGER REFERENCES owners(id);
+
+
+-- RELATIONSHIPS
+CREATE TABLE vets (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255),
+  age INTEGER,
+  date_of_graduation DATE
+);
+
+CREATE TABLE specializations (
+  id SERIAL PRIMARY KEY,
+  vet_id INTEGER REFERENCES vets(id),
+  species_id INTEGER REFERENCES species(id)
+);
+
```

Comment:
Well done overall 💯  but you must also add a new join table with the name `visits` with the following columns:

```
   CREATE TABLE visits (
   animal_id int,
   vet_id int,
   visit_date date -- another name is acceptable
   );
```

You seem to already have it as you're using it in other files but the command to create it should also be included here. 😉